bin/next-build-tools.js

```diff
@@ -136,6 +137,20 @@ program
 		});
 
 	program
+		.command('deploy-vcl [folder]')
```

Comment:
No - it supports a whole folder.  The code is taken from fastly-deploy which uploads a bunch of .vcl